pkg/sdk/go/channels_test.go

```diff
@@ -67,6 +71,20 @@ func TestCreateChannel(t *testing.T) {
 			err:     nil,
 			empty:   false,
 		},
+		{
+			desc:   "create a new channel with external UUID",
+			channel: channel1,
+			token:   token,
+			err:     nil,
+			empty:   false,
+		},
+		{
+			desc:   "create a new channel with wrong external UUID",
+			channel: chWithWrongExtID,
```

Comment:
Maybe you can short this name to `chWrongExtID`

Reply:
Done